installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc

```diff
@@ -0,0 +1,82 @@
+:_content-type: ASSEMBLY
+[id="installing-ibm-cloud-restricted"]
+= Installing a cluster on IBM Cloud VPC in a restricted network
+include::_attributes/common-attributes.adoc[]
+:context: installing-ibm-cloud-restricted
+
+toc::[]
+
+In {product-title} {product-version}, you can install a cluster in a restricted network by creating an internal mirror of the installation release content that is accessible to an existing Virtual Private Cloud (VPC) on {ibm-cloud-name}.
+
+[id="prerequisites_installing-ibm-cloud-restricted"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc#installing-ibm-cloud-account[configured an IBM Cloud account] to host the cluster.
+* You have a container image registry, such as Red Hat Quay. If you do not already have a registry, you can create a mirror registry using  xref:../../installing/disconnected_install/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[_mirror registry for Red Hat OpenShift_].
+* You have used the xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[oc-mirror OpenShift CLI (oc) plugin] to mirror the contents of the {product-registry} and and the installation media.
++
+[NOTE]
+====
+Because the installation media is on the mirror host, you can use that computer to complete all installation steps.
```

Comment:
Once overriding the private endpoints the install process, the installer has to be able to access the private endpoints from the host where installer is running for validation, so that means it is requirements to ensure installer can access these private endpoints.

Reply:
If I am following you, I believe my latest updates to  About installations in restricted networks > Required internet access and an installation host [1] should address your concern here.

[1]  https://github.com/openshift/openshift-docs/pull/55526/commits/1db5205bc4277baf39523a3bd249b4d7d2a62ea1#diff-0995ccf569e68d8a5e211b07d6f6bd89d741f4761fb517528c10bf7b445f9593R84

Reply:
Can we add one more item under "Required internet access and an installation host" and state:

* Use the installation program to access the overriding private endpoints.

Reply:
IMO I believe that we are covered when we state internet access is required to use the installation program to create the installation file. Users are instructed to use private endpoints as part of that procedure [1]. Is there a reason that you believe it should be explicitly called out?

[1] https://github.com/openshift/openshift-docs/commit/1db5205bc4277baf39523a3bd249b4d7d2a62ea1#diff-aa2bf98d200d7bbc6fbc805b89a39df30e54112c533f035e025e1160cfeeecd9R607

Reply:
On IBMCloud, once any endpoint is overridden by a private url, the installation program has to be run inside VPC, even it is not a OCP installation in restricted network.

The explicit description may help users understand. If you think the current description, I am okay with it, too,